Doc/physics_manual/sixphys.tex

```diff
@@ -948,6 +948,49 @@ \subsection{AC-dipole}
         \end{align}
 \end{enumerate}
 
+\subsection{Electron lens}
+\label{elense}
+\subsection{Hollow electron lens - uniform annular profile}
```

Comment:
thanks for spotting! Changed it!

I would also propose to proceed without the thck6d example. I think it 
will still take a bit until Riccardo has a look and we can still do it 
later. Otherwise everything should be ready?

Miriam

On 2016-07-11 10:35, Kyrre Ness Sjøbæk wrote:
>
> In Doc/physics_manual/sixphys.tex 
> <https://github.com/SixTrack/SixTrack/pull/81#discussion_r70281410>:
>
> > @@ -948,6 +948,49 @@ \subsection{AC-dipole}
> >          \end{align}
> >  \end{enumerate}
> >
> > +\subsection{Electron lens}
> > +\label{elense}
> > +\subsection{Hollow electron lens - uniform annular profile}
>
> Would be better if this was \subsubsection?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/SixTrack/SixTrack/pull/81/files/77f425ae91b875604b17ab8e181ac6b7c90190f0..f53cf930dc2f5ed974b17de3b13e543b9c292e1e#r70281410>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AIKzcYiS6-O4krc1zWQ0_UBQtjfolvwFks5qUmKxgaJpZM4I6PV7>.
>